modules/openglqt/include/modules/openglqt/canvasqt.h

```diff
@@ -83,8 +88,12 @@ class CanvasQt : public CanvasQOpenGLWidget {
     bool mapPanTriggered(QPanGesture*);
     bool mapPinchTriggered(QPinchGesture* e);
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
     //! Links QTouchDevice to inviwo::TouchDevice
+    std::map<const QPointingDevice*, TouchDevice> devices_;
```

Comment:
Should be named the same

Reply:
I want a different name so I don't miss any instance of it